inst/WORDLIST

```diff
@@ -280,6 +283,7 @@ tagrelease
 tcl
 tgt
 th
+tidyr
```

Comment:
Great to see you adding words to the WORDLIST to make your life easier in the future! This is a practice I need to do :)